terraform/panther_gcs_transport_type_infra/README.md

```diff
@@ -11,5 +11,5 @@ terraform plan
 terraform apply
 
 # or apply the change with a variables file.
-terraform apply -var-file="testing.tfvars"
+terraform apply -var-file="production.tfvars"
```

Comment:
Does this have to be named `production`? Could it be just e.g. `panther.tfvars`? 
Thinking that this could confuse customers because they might just want to send logs from Dev/Beta/Gamma/Staging to Panther

Reply:
Hmm, good point.

Reply:
Done